### PR TITLE
Refactor capacity

### PR DESCRIPTION
**Pull request by stonezdj**
Move dependency to admin server to core api.

### CODE REVIEW

File: src/core/api/systeminfo.go [hunk 1]
```diff
@@ -120,7 +122,8 @@ func (sia *SystemInfoAPI) validate() {
 func (sia *SystemInfoAPI) GetVolumeInfo() {
 	sia.validate()
 
-	capacity, err := config.AdminserverClient.Capacity()
+	systeminfo.Init()
```
Comment (cd1989) [inline src/core/api/systeminfo.go]: Please remove capacity related code from adminserver to core. One problem is you are relying env (e.g. `IMAGE_STORE_PATH`) of adminserver.

Comment (stonezdj) [reply src/core/api/systeminfo.go]: The IMAGE_STORE_PATH is not set in adminserver, it is not set in core api,  uses default value /data.

File: src/core/router.go [hunk 2]
```diff
@@ -112,6 +112,7 @@ func initRouters() {
 
 	beego.Router("/api/internal/syncregistry", &api.InternalAPI{}, "post:SyncRegistry")
 	beego.Router("/api/internal/renameadmin", &api.InternalAPI{}, "post:RenameAdmin")
+	beego.Router("/api/internal/configurations", &api.ConfigAPI{}, "get:GetInternalConfig")
```
Comment (steven-zou) [inline src/core/router.go]: Will this be a formal route in future?

Comment (stonezdj) [reply src/core/router.go]: It is not the final version, introduce /api/internal/configurations is used only for remove dependency and remove admin container.

**Pull request merged**